### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by munnokd** [The-Lively-Developers-Community/become-a-member#11] [General]
Hey guys how are you. I am Kalp Prajapati. I am software developer and open source contributor. Please give me your introduction so we know each others.

**UPDATE** Please introduce yourselves here: https://github.com/The-Lively-Developers-Community/introductions by creating a PR editing README.md.

**Comment by tirthanprajapati** [comment]
Hey this is Tirthan Prajapati, a sophomore at Vit-B interested in computer science

**Comment by KendallDoesCoding** [comment]
Hey, This is Kendall, a passionate Web Developer. I also work on Python, etc.

**Comment by SohamRoy-01** [comment]
Hey , I am Soham passionate about full stack developer but right not frontend developer is my forte...
Hope we can bring some unique ideas and projects in the community...and can contribute in a wider space..

**Comment by munnokd** [comment]
Can we start a new project? I can give my contribution into that project.

**Reply by KendallDoesCoding** [reply]
I had a idea! I want to make a Repo and Website called Intro To Coding & Choosing the Right Language which will be a documentation to help people begin coding!

**Reply by KendallDoesCoding** [reply]
Our team, is thinking of ideas. We want to sort out our website domain first.

**Reply by SohamRoy-01** [reply]
### How is CodeSpectrum.io

**Reply by KendallDoesCoding** [reply]
@SohamRoy-01 Unable to access to site.

**Comment by KendallDoesCoding** [comment]
Please introduce yourselves here: https://github.com/The-Lively-Developers-Community/introductions by creating a PR editing README.md.. Thanks

**Reply by munnokd** [reply]
ok
